docs/shared/ungrib.yaml

```diff
@@ -1,9 +1,16 @@
+platform:
+  account: me
+  scheduler: slurm
```

Comment:
Looking at [the docs PR](https://github.com/ufs-community/uwtools/pull/442/files), I think it's actually nice if the `platform` block appears in the YAML shown in the CLI section, because it's a more realistic example. So I think this is a good addition, but maybe we need a simple statement in the UW YAML section where this shared snippet is also pulled into the RST doc to just mention that `platform` is described elsewhere. Any maybe we should add this same `platform` block to the other `shared/*.yaml` files and add the same statement to their UW YAML sections for consistency?

Reply:
On yet further inspection, I think the UW YAML pages (e.g. [this one](https://uwtools.readthedocs.io/en/main/sections/user_guide/yaml/components/chgres_cube.html#chgres-cube-yaml)) already have good language for the `platform` block, including a link to its description. So maybe we just need to add this `platform` block to the other `shared/*.yaml` files for consistency.

Reply:
I can add the block to the other yamls and let @WeirAE add the appropriate verbiage in PR #442.

Reply:
Verbiage all set